app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -245,4 +248,14 @@ public static String unescapeString(final String input) {
 
         return builder.toString();
     }
+
+    public static String getPath(final Document document) {
```

Comment:
please don't add markor.Document to StringUtils if not required, which is not here.  
For example, have it in DocumentIO or at the other getPath() place